### PR TITLE
Upgrade to 3.0.2.0 and add run-exports

### PR DESCRIPTION
**Pull request by saraedum**
and some minor maintenance

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

**Comment by saraedum** [comment]
@isuruf, I think this one is ready.

File: recipe/0001-Adapt-for-Windows-build.patch [hunk 1]
```diff
@@ -0,0 +1,58 @@
+From 091a3edbd59bb5601df2acffb698849fb807193d Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
```
Comment (isuruf) [inline recipe/0001-Adapt-for-Windows-build.patch]: Keep both our name/email here?

Comment (saraedum) [reply recipe/0001-Adapt-for-Windows-build.patch]: Sorry, I guess you're the original author.

Comment (saraedum) [reply recipe/0001-Adapt-for-Windows-build.patch]:
```suggestion
```

**Pull request merged**